src/components/templates/AnimateMap/AnimateMap.tsx

```diff
@@ -0,0 +1,85 @@
+import * as PIXI from "pixi.js";
+
+import React, { useState, useRef, useEffect } from "react";
+import { Provider, ReactReduxContext } from "react-redux";
+import { AppConsumer, Container, Stage, useApp } from "@inlet/react-pixi";
+import { AnimateMapVenue } from "../../../types/venues";
+import { useUser } from "hooks/useUser";
+import { useSelector } from "../../../hooks/useSelector";
+import { animateMapStageOptionsSelector } from "../../../utils/selectors";
+
+import "./AnimateMap.scss";
+
+import { MapContainer } from "./components/Map/MapContainer";
+import { MAP_IMAGE } from "./constants/Resources";
+
+export interface AnimateMapProps {
+  venue: AnimateMapVenue;
+}
+
+export const AnimateMap: React.FC<AnimateMapProps> = () => {
+  const options = useSelector(animateMapStageOptionsSelector);
+
+  const containerRef = useRef<HTMLDivElement>(null);
+  const { user, profile } = useUser();
+
+  if (!user || !profile) {
+    return <>Loading..</>;
+  }
+
+  const container = containerRef.current ?? null;
+
+  return (
+    <div ref={containerRef} className="animatemap-venue-container">
+      {container && (
+        <ReactReduxContext.Consumer>
+          {({ store }) => (
+            <Stage
+              width={container.offsetWidth}
+              height={container.offsetHeight}
+              options={Object.assign(options, { resizeTo: container })}
+            >
+              <AppConsumer>
+                {(app) => (
+                  <Provider store={store}>
+                    {app ? <LoadingContainer /> : <Container />}
+                  </Provider>
```

Comment:
Please use the redux hooks-based API: https://react-redux.js.org/api/hooks

Reply:
See this for potentially related reason why this isn't currently using hooks: https://github.com/sparkletown/sparkle/pull/1617/files/6340eaee9d3913df6b50ab525569bfa35da9e5ea#r656036377